frontend/app/.server/domain/dtos/client-eligibility.dto.ts

```diff
@@ -8,3 +8,7 @@ export type ClientEligibilityDto = Readonly<{
     isEligible: boolean;
   }>;
 }>;
+
+export type ClientEligibilityRequestDto = Readonly<{
+  clientIdentification: string;
```

Comment:
Can be shorten to `clientId`.

```suggestion
  clientId: string;
```